components/bluetooth/lib.rs

```diff
@@ -924,4 +927,12 @@ impl BluetoothManager {
         // TODO: Implement this when supported in lower level
         return Err(BluetoothError::NotSupported);
     }
+
+    // https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetooth-getavailability
+    fn get_availability(&mut self) -> BluetoothResponseResult {
```

Comment:
This is called via an async promise and the handle function requires a result, because usually a promise could fail, but this will only resolve. [Step 2 - 3.](https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetooth-getavailability)